reporting_app/static/library_preps.js

```diff
@@ -0,0 +1,207 @@
+var heatmap_x_coords = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
+
+metrics = {
+    'pc_q30': {name: '%Q30', path: ['bioinformatics_qc', 'aggregated', 'pc_q30']},
+    'pc_mapped_reads': {name: '% mapped reads', path: ['bioinformatics_qc', 'aggregated', 'pc_mapped_reads']},
+    'species': {name: 'Species', path: ['bioinformatics_qc', 'species_name'], type: 'category'},
+    'matching_species': {name: 'Matching species', path: ['bioinformatics_qc', 'aggregated', 'matching_species'], type: 'category'},
+    'het_hom_ratio': {name: 'Het/hom ratio', path: ['bioinformatics_qc', 'sample_contamination', 'het_hom_ratio']},
+    'freemix': {name: 'Freemix', path: ['bioinformatics_qc', 'sample_contamination', 'freemix']},
+    'genotype_match': {name: 'Genotype match', path: ['bioinformatics_qc', 'aggregated', 'genotype_match'], type: 'category'},
+    'pc_optical_duplicates': {name: '% optical duplicates', path: ['bioinformatics_qc', 'aggregated', 'from_all_run_elements', 'pc_opt_duplicate_reads']},
+    'pc_duplicates': {name: '% duplicates', path: ['bioinformatics_qc', 'aggregated', 'from_all_run_elements', 'pc_duplicate_reads']},
+    'gc_bias_mean_dev': {name: 'GC bias (mean deviation)', path: ['bioinformatics_qc', 'aggregated', 'from_all_run_elements', 'gc_bias', 'mean_deviation']},
+    'gc_bias_slope': {name: 'GC bias (slope)', path: ['bioinformatics_qc', 'aggregated', 'from_all_run_elements', 'gc_bias', 'slope']},
+    'pc_adaptor': {name: '% adaptor', path: ['bioinformatics_qc', 'aggregated', 'from_all_run_elements', 'pc_adaptor']},
+    'mean_insert_size': {name: 'Mean insert size', path: ['bioinformatics_qc', 'aggregated', 'from_all_run_elements', 'mean_insert_size']},
+    'library_size': {name: 'Picard est. lib size', path: ['bioinformatics_qc', 'aggregated', 'from_all_run_elements', 'picard_est_lib_size']},
+    'av_conc': {name: 'Ave. Conc. (nM)', path: ['library_qc', 'Ave. Conc. (nM)']},
+    'pc_cv': {name: '%CV', path: ['library_qc', '%CV']},
+    'raw_cp': {name: 'Raw CP', path: ['library_qc', 'Raw CP']},
+    'ntp_volume': {name: 'NTP Volume (uL)', path: ['library_qc', 'NTP Volume (uL)']},
+    'adjusted_conc': {name: 'Adjusted Conc. (nM)', path: ['library_qc', 'Adjusted Conc. (nM)']},
+    'original_conc': {name: 'Original Conc. (nM)', path: ['library_qc', 'Original Conc. (nM)']},
+    'ntp_transfer_vol': {name: 'NTP Transfer Volume (uL)', path: ['library_qc', 'NTP Transfer Volume (uL)']},
+    'rsb_transfer_vol': {name: 'RSB Transfer Volume (uL)', path: ['library_qc', 'RSB Transfer Volume (uL)']},
+    'sample_transfer_vol': {name: 'Sample Transfer Volume (uL)', path: ['library_qc', 'Sample Transfer Volume (uL)']},
+    'tsp1_transfer_vol': {name: 'TSP1 Transfer Volume (uL)', path: ['library_qc', 'TSP1 Transfer Volume (uL)']},
+    'qc_flag': {name: 'QC flag', path: ['qc_flag'], type: 'category'},
+    'project_id': {name: 'Project ID', path: ['project_id'], type: 'category'}
```

Comment:
Not really a request at this point but the path could be written using the dot notation (`'bioinformatics_qc.aggregated.from_all_run_elements.pc_duplicate_reads'`) instead of the array notation